yarn-project/end-to-end/src/e2e_non_contract_account.test.ts

```diff
@@ -42,12 +30,13 @@ describe('e2e_non_contract_account', () => {
 
     // Send transaction as arbitrary non-contract account
     const nullifier = new Fr(940);
-    const receipt = await contractWithNoContractWallet.methods.emit_nullifier(nullifier).send().wait({ interval: 0.1 });
-    expect(receipt.status).toBe(TxStatus.MINED);
+    const { debugInfo } = await contractWithNoContractWallet.methods
+      .emit_nullifier(nullifier)
+      .send()
+      .wait({ interval: 0.1, debug: true });
 
-    const tx = await aztecNode!.getTx(receipt.txHash);
```

Comment:
Decided to clean up a few tests and use the nicer API (passing debug: true to wait) now that here was a diff anyway.